charts/kubewarden-defaults/README.md

```diff
@@ -0,0 +1,46 @@
+# kubewarden-defaults
+
+`kubewarden-defaults` is the Helm chart that installs a default PolicyServer
+required by the Kubewarden to run `ClusterAdmissionPolicy`. It should be installed
+before installing any policies.
+
+## Installing
+
+For example:
+```console
+$ helm repo add kubewarden https://charts.kubewarden.io
+$ helm install --create-namespace -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
+```
+
+For a more comprehensive documentation about how to install the whole Kubewarden
+stack, check the `kubewarden-controller` chart documentation out.
+
+## Upgrading the charts
+
+Please refer to the release notes of each version of the helm charts.
+These can be found [here](https://github.com/kubewarden/helm-charts/releases).
+
+## Uninstalling the charts
+
+To uninstall/delete kubewarden-crds use the following command:
+
+```console
+$ helm uninstall -n kubewarden kubewarden-defaults
+```
+
+The commands remove all the Kubernetes components associated with the chart.
+Keep in mind that the chart is required by the `kubewarden-controller` chart.
```

Comment:
Is that correct? I think this chart can be removed without affecting the `kubewarden-controller`. The removal of the chart is however going to cause all the policies running on the `default` Policy Server to be removed. This is something that should be written in big letters, both here and inside of our docs.